docs/Modpacks/Materials-and-Elements/Modifying-Existing-Materials.md

```diff
@@ -71,3 +71,15 @@ You can also add flags to existing materials:
         
     });
 ```
+
+Editing the color of an existing material:
+
+
+```js title="material_modification.js"
+    GTCEuStartupEvents.materialModification(event => { //(1)
+        GTMaterials.BismuthBronze.setMaterialARGB(0x82AD92) //(2)
+    })
+```
+
+1. You can use both the `gtceu:material` event as well as `materialModification` to modify materials from both GT and its addons. However, you cannot register materials in `materialModification`.
```

Comment:
don't say this. it'll inevitably create confusion when someone tries to use a KJS material, assuming it to work as well.